src/cloud/release-notes/backward-incompatible-changes.md

```diff
@@ -0,0 +1,37 @@
+---
+group: release-notes
+title: Backward Incompatible Changes
+---
+
+Use the following information to learn about backward incompatible changes that require you to adjust Cloud configuration and processes for existing Cloud projects when you upgrade to the latest release of the {{site.data.var.ct}} package or other {{site.data.var.csuite}} packages.
+
+### Service version requirement changes
+
+**PHP minimum version requirement**–In {{ site.data.var.ct }} v2002.1.0 and later, the minimum php version requirement is 7.1.3+. If your environment configuration specifies an earlier version, update the [php configuration]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app.html#configure-php-options) in the `.magento.app.yaml` file. 
+
+### Environment variable changes
+
+The following environment variables have been removed from the `{{site.var.data.ct}}` package:
+
+-  `SCD_EXCLUDE_THEMES`–use the `SCD_MATRIX` variable instead.
+-  `STATIC_CONTENT_THREADS`–use the `SCD_THREADS` variable the instead.
+-  `DO_DEPLOY_STATIC_CONTENT`–use the `SKIP_SCD` variable instead.
+-  `STATIC_CONTENT_SYMLINK`–Now, the `{{site.data.var.ct}}` always generates symlinks for static content in the `pub/static` directory when you generate static content during the build process.
```

Comment:
@oshmyheliuk  Please check the description for the `STATIC_CONTENT_SYMLINK` variable for technical accuracy.

Reply:
Looks good, but `create symlinks` will be more accurate than `generates symlink`